### PR TITLE
Strip whitespace from user input

### PR DESCRIPTION
**Pull request by tristanmorgan**
and changes to appease updated RuboCop.

### CODE REVIEW

File: exe/awskeyring [hunk 1]
```diff
@@ -9,7 +9,7 @@ end
 
 begin
   AwskeyringCommand.start
-rescue Keychain::UserCancelledError => err
-  warn err.to_s
+rescue Keychain::UserCancelledError => e
```
Comment (JohnVonNeumann) [inline exe/awskeyring]: I don't agree with the use of single char variable names in interpreted languages. We aren't writing C. I also don't think this is the correct place to merge these changes. I think they should be pulled into a separate PR for clarity of objective.

File: lib/awskeyring/awsapi.rb [hunk 2]
```diff
@@ -185,7 +185,7 @@ def self.region
     # @return [String] key The aws_access_key_id
     # @return [String] secret The aws_secret_access_key
     # @return [String] account the associated account name.
-    def self.rotate(account:, key:, secret:, key_message:) # rubocop:disable  Metrics/MethodLength, Metrics/AbcSize
+    def self.rotate(account:, key:, secret:, key_message:) # rubocop:disable  Metrics/MethodLength
```
Comment (JohnVonNeumann) [inline lib/awskeyring/awsapi.rb]: Please add comments around motivations for ignoring rubocop directives.

File: lib/awskeyring_command.rb [hunk 3]
```diff
@@ -457,7 +457,7 @@ def ask_check(existing:, message:, secure: false, optional: false, validator: ni
   end
 
   def ask_missing(existing:, message:, secure: false, optional: false)
-    existing || ask(message: message, secure: secure, optional: optional)
+    existing || ask(message: message, secure: secure, optional: optional).strip
```
Comment (JohnVonNeumann) [inline lib/awskeyring_command.rb]: LGTM.

File: lib/awskeyring/awsapi.rb (hunk 2 above)
Comment (tristanmorgan) [reply lib/awskeyring/awsapi.rb]: I am trying to have the minimal exceptions to the default RuboCop rules. this exception was no longer needed.

File: exe/awskeyring (hunk 1 above)
Comment (tristanmorgan) [reply exe/awskeyring]: from https://github.com/rubocop-hq/rubocop/pull/6460 the default name is "e", I'm not fussed as to what its called and I am going to go with the default.

Comment (JohnVonNeumann) [reply exe/awskeyring]: Fair call, will defer to the wizards of linting here.

File: lib/awskeyring/awsapi.rb (hunk 2 above)
Comment (JohnVonNeumann) [reply lib/awskeyring/awsapi.rb]: Sorry, I misread here, I thought you were adding an ignore rule. All good.

**Comment by JohnVonNeumann** [comment]
LGTM.
